libra-node/src/main_node.rs

```diff
@@ -164,19 +168,50 @@ pub fn setup_network(
             .take_private()
             .expect("identity key should be present");
 
+        // TODO(philiphayes): seed peers should be fully rendered. then we can
```

Comment:
makes me think that we have a bunch of `TODO(abhayb)` as well, not sure who is taking care of them?

Reply:
i think abhay mostly did plain `TODO:` : D